src/main/java/one/oktw/ModConfig.java

```diff
@@ -8,6 +8,7 @@
 @Config(name = "FabricProxy")
 public class ModConfig implements ConfigData {
     private Boolean BungeeCord = false;
+    private Boolean BungeeCordWorkaround = false;
```

Comment:
I think it should be enabled by default
```suggestion
    private Boolean BungeeCordWorkaround = true;
```